assets/js/translate-analytics.js

```diff
@@ -0,0 +1,54 @@
+// Detects if the page is translated and notifies analyticis
+
+// adapted from: https://analytical42.com/2022/detect-track-translations-ga4/
+// This code does its best to detect if the user changes the language of the page.
+// Right now it fully supports logging from users changing the page language from the website itself regardless of the browser used
+// This struggles with detecting and reporting the new language when the user uses the browser to
+// change the language.
+// Chrome - Full support, we can detect when the user translates the page, and what language the page is translated to
+// Safari - Full support, we can detect when the user translates the page, and what language the page is translated to
+// Edge - Half support, we can detect whne the user translates the page, just not what language they changed it to
+// Firefox - No support, we cannot detect when the user translates the page
```

Comment:
It seems to work in Firefox for me (I can add a console.log to this and see the changes reflected)? Is it that it doesn't work in a specific scenario? I like this elaborate note here, but it needs to be accurate.

Reply:
Okay, I will update the comment.  I it appears that Firefox must have updated since this was written.  As initial testing Firefox did nothing to modify the `html` or `title`